Dockerfile-timescaledb

```diff
@@ -50,7 +49,7 @@ RUN curl https://haproxy.debian.net/bernat.debian.org.gpg \
     | gpg --dearmor > /usr/share/keyrings/haproxy.debian.net.gpg
 
 RUN echo deb "[signed-by=/usr/share/keyrings/haproxy.debian.net.gpg]" \
-    http://haproxy.debian.net bookworm-backports-${HAPROXY_VERSION} main \
+    http://haproxy.debian.net $(. /etc/os-release && echo "$VERSION_CODENAME")-backports-${HAPROXY_VERSION} main \
     > /etc/apt/sources.list.d/haproxy.list
 
 RUN apt-get update && apt-get install --no-install-recommends -y \
```

Comment:
Do you think we could install timescale to a pinned version parameterized by a docker arg

e.g.

```
ARG TIMESCALE_VERSION=2.13.1


apt install timescaledb-2-$TIMESCALE_VERSION-postgresql-$PG_MAGOR_VERSION_$TIMESCALE_VERSION~ubuntu(. /etc/os-release && echo "$VERSION_ID")_amd64.deb
```

Currently we just take the latest but if the latest changes this image won't update as that docker layer is already cached to a specific version.

Currently when I spin up a new fly app I see this version for TS:

![image](https://github.com/fly-apps/postgres-flex/assets/2615540/c305f02d-016a-4e43-ab5e-9e7de4bdd325)

But latest on package cloud is 2.13.

![image](https://github.com/fly-apps/postgres-flex/assets/2615540/13926e64-22bf-483c-8a3e-0c3778dcde9a)

Reply:
Yes, we can. Shall we make that change?

Reply:
That would be fantastic.

Reply:
@JAForbes Done 🙂